tests/gpu_dfcc/gpu_dfcc1/input.dat

```diff
@@ -0,0 +1,34 @@
+#! cc-pvdz (H2O)2 Test DF-CCSD vs GPU-DF-CCSD
+
+sys.path.insert(0,'../../../.')
+
+import gpu_dfcc
+
+molecule {
+           O          0.000000000000     0.000000000000    -0.068516219310   
+           H          0.000000000000    -0.790689573744     0.543701060724   
+           H          0.000000000000     0.790689573744     0.543701060724   
+}
+
+memory 32000 mb
```

Comment:
What happens on a computer with only CPUs? Do we need to disable if hardware not detected? Only import the plugin if `num_gpus>1`? Or is it just harmless or non-obliging if no GPUs to work with? Clearly I've just no experience with hardware-dependent tests.

In any case, I hope memory req. can be toned down?